setup.py

```diff
@@ -0,0 +1,11 @@
+import os
+from distutils.core import setup
+
+setup(
+    name = 'svgdigitizer',
+    version = '0.1.0',
+    packages = ['svgdigitizer',],
+    license = 'GPL 3.0+',
+    long_description = open('README.md').read(),
+    include_package_data=True,
+)
```

Comment:
With this, it should be possible to `pip install .` the svgdigitizer. Once we published it to PyPI, it should be possible to `pip install svgdigitizer`.